python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1187,6 +1187,8 @@ def my_dbt_op(dbt: DbtCliResource):
             "DAGSTER_DBT_CLI": "true",
             # Run dbt with unbuffered output.
             "PYTHONUNBUFFERED": "1",
+            # Allow IO streaming when running in Windows.
+            "PYTHONLEGACYWINDOWSSTDIO": "1",
```

Comment:
is this possible to override? Since it's listed after `**os.environ.copy()` it will take precedence right

Reply:
Probably no one will need to but would be nice in case there's an unintended consequence

Reply:
Good point